src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -452,6 +452,8 @@ private int WriteRunToErrorList(Run run, string logFilePath)
 
             (dataCache.SarifErrors as List<SarifErrorListItem>).AddRange(sarifErrors);
             SarifTableDataSource.Instance.AddErrors(sarifErrors);
+            SarifLocationTagger.NotifyAllTagsChanged();
```

Comment:
Comment why this is here..

This causes already open "text views" to be tagged when SARIF logs are processed after a view is opened.